### DISCUSSION TITLE
Will objective-c realm be EOLed?

### DISCUSSION THREAD

**Discussion by timothycosta** [realm/realm-cocoa#7730] [General]
I noticed today that `realm-cocoa` now redirects to `realm-swift`. I see that the latest release still has an objective-c version, but I'm wondering if it will continue to receive updates?

Thanks!

**Comment by kowongh** [comment]
I'm wondering about this as well. I was still able to download the latest realm-objc from Swift Package Manager but it doesn't seem like it's being supported anymore. I can't find any documentation or discussion about this.

For legacy projects do we have to add dependencies for both realm-objc and realm-swift?
